tdwii_plus_examples/nevent_sender.py

```diff
@@ -104,10 +104,12 @@ def _setup_argparser():
     # Description
     parser = argparse.ArgumentParser(
         description=(
-            "The neventscu application implements a Service Class User "
```

Comment:
I understand what you mean as in main, when the association is created, it is not using the role selection negotiation to set the SCP role for the Unified Procedure Step - Event SOP Class to SCP and the SCU. So, as explained in KK.1.3.1 Association Negotiation, the default SCU role for an association-requestor will be used.
I think that an association-requestor could actually propose the UPS Event presentation context as SCP and that the association-acceptor could accept it with the role SCU. Whether pynetdicom alllows this, I am not sure of. But if it does it would be more in line with the standard.

Reply:
I will update the content to reflect sender vs scu
The role proposal you describe is an option in the Standard but that isn't what the example is trying to demonstrate.
I think the approach you mention is worthy of a longer discussion in a separate issue.

Reply:
> I understand what you mean as in main, when the association is created, it is not using the role selection negotiation to set the SCP role for the Unified Procedure Step - Event SOP Class to SCP and the SCU. So, as explained in KK.1.3.1 Association Negotiation, the default SCU role for an association-requestor will be used.
> I think that an association-requestor could actually propose the UPS Event presentation context as SCP and that the association-acceptor could accept it with the role SCU. Whether pynetdicom alllows this, I am not sure of. But if it does it would be more in line with the standard. 

Sorry, I didn't realise you were still reviewing and I would not have merged had i realised that.  I will follow up with a "clean up" PR to address any remaining comments